feg/cloud/configs/service_registry.yml

```diff
@@ -36,6 +36,8 @@ services:
         port: 9103
       session_proxy:
         port: 9079
+      n7_n40_proxy:
```

Comment:
you don't have n7 n40 proxy router itself, I don't think - this mapping does anything/needed

Reply:
Per what we discussed with @emakeev  we may need the router since we will have to check request parametrs (same router check IMSI on a request)